eq-author/src/App/questionPage/Preview/index.js

```diff
@@ -64,25 +60,13 @@ export const UnwrappedPreviewPageRoute = ({ loading, data }) => {
   if (loading) {
     return <Loading height="38rem">Preview loading…</Loading>;
   }
-
-  /*  eslint-disable react/no-danger */
   const { questionPage } = data;
-  const { description, guidance, answers } = questionPage;
-
-  let title = questionPage.title.replace(/(<p[^>]+?>|<p>|<\/p>)/gim, "");
+  const { title, description, guidance, answers } = questionPage;
 
   return (
     <EditorLayout page={questionPage} preview routing>
       <Container>
-        <PageTitle>
-          {title ? (
-            <div dangerouslySetInnerHTML={{ __html: title }} />
-          ) : (
-            <Error data-test="no-title" large>
-              Missing Page Title
-            </Error>
-          )}
-        </PageTitle>
+        <PageTitle title={title} />
```

Comment:
If we're turning this into a component, can it use `children` instead?

Reply:
It doesn't make sense in this case: 

```
const PageTitle = ({ title }) => {
  let pageTitle = title && title.replace(/(<p[^>]+?>|<p>|<\/p>)/gim, "");
  return (
    <Title>
      {pageTitle ? (
        /*  eslint-disable-next-line react/no-danger */
        <div dangerouslySetInnerHTML={{ __html: pageTitle }} />
      ) : (
        <Error data-test="no-title" large>
          Missing Page Title
        </Error>
      )}
    </Title>
  );
};

```